src/components/Icon.jsx

```diff
@@ -32,11 +32,21 @@ const SortIcon = () => {
     </div>
   );
 };
+const CloseIcon = () => {

```

Comment:
At the moment; the close icon can onlu be used when width and height is 14.
Consider including `{props}` as an arg to svg icons so that users can pass in other properties to edit the icon; making it very reusable.

Then spread it in  `<svg...>`
e.g.
`<svg width="14" height="14" viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>`